packages/ProgressBar/src/ProgressBar.js

```diff
@@ -7,39 +7,54 @@ import * as sc from "./ProgressBar.styles";
 const propTypes = {
   /** Descriptive a11y text for assistive technologies. */
   a11yText: PropTypes.string,
+
   /** Description for the ProgressBar */
   bodyText: PropTypes.string,
-  /** Header text for the ProgressBar */
-  header: PropTypes.string,
+
   /** Specifies how much progress has been completed from 0-100 */
   completed: PropTypes.number,
+
+  /** Text for the heading displayed above ProgressBar */
+  header: PropTypes.node,
+
+  /** Semantic heading level of header */
+  headerLevel: PropTypes.oneOf[(1, 2, 3, 4, 5, 6)],
+
+  /** If ProgressBar is displayed in a compact style */
+  isCompact: PropTypes.bool,
 };
 
 const defaultProps = {
   a11yText: null,
   bodyText: null,
-  header: null,
   completed: 0,
+  header: null,
+  headerLevel: 3,
+  isCompact: false,
 };
 
 function ProgressBar(props) {
-  const { a11yText, header, bodyText, completed, ...moreProps } = props;
-  const bestAria = a11yText || "Loading";
+  const { a11yText, bodyText, completed, header, headerLevel, isCompact, ...moreProps } = props;
+
   return (
     <sc.ProgressBar {...moreProps}>
-      <Heading level={3}>{header}</Heading>
-      <sc.Bar>
+      {header ? (
```

Comment:
could do `header && ...` but no big deal. same a few lines down where you have the `?`. This gets rid of the `null`s

Reply:
Yeah, I've been deliberately trying to avoid `&&` for conditional rendering... there are some cases it can bite you:

https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx